oracle_cardano/processor/tx_processors/success/bridging_requested_processor_skyline.go

```diff
@@ -0,0 +1,345 @@
+}
+
+/*
+func (*BridgingRequestedProcessorSkylineImpl) addRefundRequestClaim(
+	claims *core.BridgeClaims, tx *core.CardanoTx, metadata *common.BridgingRequestMetadata,
+) {
+
+		var outputUtxos []core.Utxo
+		for _, output := range tx.Outputs {
+			outputUtxos = append(outputUtxos, core.Utxo{
+				Address: output.Address,
+				Amount:  output.Amount,
+			})
+		}
+
+		// what goes into UtxoTransaction
+		claim := core.RefundRequestClaim{
+			TxHash:             tx.Hash,
+			RetryCounter:       0,
+			RefundToAddress:    metadata.SenderAddr,
+			DestinationChainId: metadata.DestinationChainId,
+			OutputUtxos:        outputUtxos,
+			UtxoTransaction:    core.UtxoTransaction{},
+		}
+
+		claims.RefundRequest = append(claims.RefundRequest, claim)
+
+		p.logger.Info("Added RefundRequestClaim",
+		"txHash", tx.Hash, "metadata", metadata, "claim", core.RefundRequestClaimString(claim))
+}
+*/
+
+func (p *BridgingRequestedProcessorSkylineImpl) validate(
+	tx *core.CardanoTx, metadata *common.BridgingRequestMetadata, appConfig *cCore.AppConfig,
+) error {
+	chainConfig := appConfig.CardanoChains[tx.OriginChainID]
+	if chainConfig == nil {
+		return fmt.Errorf("unsupported chain id found in tx. chain id: %v", tx.OriginChainID)
+	}
+
+	if err := utils.ValidateOutputsHaveTokens(tx, appConfig); err != nil {
+		return err
+	}
+
+	multisigUtxo, err := utils.ValidateTxOutputs(tx, appConfig, false)
+	if err != nil {
+		return err
+	}
+
+	cardanoSrcConfig, _ := cUtils.GetChainConfig(appConfig, tx.OriginChainID)
+	if cardanoSrcConfig == nil {
+		return fmt.Errorf("origin chain not registered: %v", tx.OriginChainID)
+	}
+
+	cardanoDestConfig, ethDestConfig := cUtils.GetChainConfig(appConfig, metadata.DestinationChainID)
+	if cardanoDestConfig == nil && ethDestConfig == nil {
+		return fmt.Errorf("destination chain not registered: %v", metadata.DestinationChainID)
+	}
+
+	if len(metadata.Transactions) > appConfig.BridgingSettings.MaxReceiversPerBridgingRequest {
+		return fmt.Errorf("number of receivers in metadata greater than maximum allowed - no: %v, max: %v, metadata: %v",
+			len(metadata.Transactions), appConfig.BridgingSettings.MaxReceiversPerBridgingRequest, metadata)
+	}
+
+	receiverAmountSum := big.NewInt(0)
+	feeSum := uint64(0)
+	foundAUtxoValueBelowMinimumValue := false
+	foundAnInvalidReceiverAddr := false
+
+	exchangeRate, err := GetExchangeRate(metadata.DestinationChainID)
+	if err != nil {
+		return err
+	}
+
+	for _, receiver := range metadata.Transactions {
+		receiverAddr := strings.Join(receiver.Address, "")
+
+		var totalAmountToBridge uint64
+
+		if cardanoDestConfig != nil {
+			if receiver.Additional == nil {
+				if !cardanotx.IsValidOutputAddress(receiverAddr, cardanoDestConfig.NetworkID) {
+					foundAnInvalidReceiverAddr = true
+
+					break
+				}
+
+				if receiverAddr == cardanoDestConfig.BridgingAddresses.FeeAddress {
+					feeSum += receiver.Amount
+
+					continue
+				} else {
+					foundAUtxoValueBelowMinimumValue = true
```

Comment:
I dont understand. this is not correct. it doesnt mean that if additional == nil && receiverAddr != feeAddr, that it is utxoBelowMinValue